### DISCUSSION TITLE
Focus state not wrapping around the whole icon

### DISCUSSION THREAD

**Discussion by pradyuk** [twilio-labs/paste#795] [Q&A] [answered]
Hi team,

Im trying to use the branding icon, but the focus state doesn't seem to wrap around perfectly. I would appreciate some help.

<img width="150" alt="Screen Shot 2020-09-24 at 6 49 31 PM" src="https://user-images.githubusercontent.com/11803735/94217444-de848e00-fe96-11ea-95d1-79659be81d0f.png">

**Comment by serifluous** [comment]
Hey @pradyuk ! Is there a prototype or codesandbox you could share with us? It‘ll make debugging this a little easier.

**Reply by pradyuk** [reply]
Hey @serifluous! Thanks for your quick turnaround. Would a PR to my code work? If you have access, storybook is configured which might help you. [Link to PR](https://code.hq.twilio.com/twilio/console/pull/91)

**Reply by richbachman** [reply]
Hi @pradyuk you may need to give your `Anchor` a `display="inline-block"` or `display="block"`, depending on your use-case. That should help with the focus state.

**Reply by pradyuk** [reply]
@richbachman thanks for that, it definitely looks much better. The logo is still not centered though, can you suggest any workaround for that? I pasted the image below.

**Reply by richbachman** [reply]
@pradyuk would you mind pushing your latest, so I can see the most recent deployment. That will help with debugging. Thanks.

**Reply by pradyuk** [reply]
@richbachman done.

**Comment by pradyuk** [comment]
![Screen Shot 2020-09-24 at 8 15 41 PM](https://user-images.githubusercontent.com/11803735/94222617-c0bd2600-fea2-11ea-848a-629db1c2ced3.png)

**Comment by richbachman** [comment, marked as answer]
@pradyuk after taking a closer look, it looks like you might be running an old version of the icons package. I'm seeing the icon with a `display: inline-flex` style. That throws off the height and along with it the focus state. If you update your icon package,  you should see a `display: block` style instead. That should fix the issue.

Icons also take a `display` prop so if updating the package is risky, you could also just pass the icon `display="block"`.

You can find the latest Paste package versions here: https://paste.twilio.design/getting-started/engineering#install-paste-system-packages

We're working on a way to make keeping these up to date much easier as well. More on that soon.

**Reply by pradyuk** [reply]
@richbachman Thanks so much, this was super helpful and worked like a charm!
